sdk/core/core-rest-pipeline/test/node/pipeline.spec.ts

```diff
@@ -1,19 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as http from "https";
-import * as https from "https";
+import http from "https";
```

Comment:
is the `* as foo` syntax just not required anymore?

Reply:
I am not sure about the root cause yet, but `sinon` cannot mock `import * as` and gave this error: "ES Modules cannot be stubbed".  I had to change the import in the src files as well, otherwise, sinon is not able to modify http.request.